Makefile

```diff
@@ -1,136 +1,102 @@
 TERRAFILE_VERSION=0.8
 ARM_TEMPLATE_TAG=1.1.6
 RG_TAGS={"Product" : "Get Schools Experience"}
-SERVICE_SHORT=gse
 REGION=UK South
-
-ifndef VERBOSE
-.SILENT:
-endif
-
-help:
-	echo "Secrets:"
-	echo "  This makefile gives the user the ability to safely display and edit azure secrets which are used by this project. "
-	echo ""
-	echo "Commands:"
-	echo "  edit-app-secrets  - Edit Application specific Secrets."
-	echo "  print-app-secrets - Display Application specific Secrets."
-	echo ""
-	echo "Parameters:"
-	echo "All commands take the parameter development|review|test|production"
-	echo ""
-	echo "Examples:"
-	echo ""
-	echo "To edit the Application secrets for Development"
-	echo "        make  development edit-app-secrets"
-	echo ""
-
-APPLICATION_SECRETS=SE-SECRETS
-INFRA_SECRETS=SE-INFRA-SECRETS
+SERVICE_NAME=get-schools-experience
+SERVICE_SHORT=gse
+DOCKER_REPOSITORY=ghcr.io/dfe-digital/schools-experience
 
 .PHONY: development
 development:
```

Comment:
Can it be renamed to development_aks so it doesn't clash with the original "development" paas deployment?

Reply:
done